cmake/libs/Libbacktrace.cmake

```diff
@@ -14,14 +14,39 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
+# On MacOS, the default C compiler (/usr/bin/cc) is actually a small script that dispatches to a
+# compiler the default SDK (usually /Library/Developer/CommandLineTools/usr/bin/ or
```

Comment:
On MacOS, the default C compiler (/usr/bin/cc) is script that dispatches to the compiler
provided by the default SDK (usually /Library/Developer/CommandLineTools/usr/bin/ or